server/config/persist_options.go

```diff
@@ -245,7 +245,7 @@ func (o *PersistOptions) GetMaxMergeRegionSize() uint64 {
 
 // GetMaxMergeRegionKeys returns the max number of keys.
 func (o *PersistOptions) GetMaxMergeRegionKeys() uint64 {
-	return o.getTTLUintOr(maxMergeRegionKeysKey, o.GetScheduleConfig().MaxMergeRegionKeys)
+	return o.getTTLUintOr(maxMergeRegionKeysKey, o.GetScheduleConfig().GetMaxMergeRegionKeys())
```

Comment:
Do we need to make sure the `MaxMergeRegionKeys` will be `MaxMergeRegionSize * 10000` when `MaxMergeRegionKeys` is set to 0 with TTL?

Reply:
yes, it will return size * 10000 if `MaxMergeRegionKeys ` is not exist .